e2e_samples/parking_sensors/adf/factory/mdwdops-adf-dev-16.json

```diff
@@ -0,0 +1,9 @@
+{
+	"name": "mdwdops-adf-dev-16",
+	"location": "australiaeast",
+	"identity": {
+		"type": "SystemAssigned",
+		"principalId": "3f1a4571-2be2-4d80-9b62-6901a40b69d5",
```

Comment:
Should we consider not commit this 2 Ids on the repo? I know they are not secrets, but as a general best practice maybe we can avoid to have them commited.

Reply:
Yeah, technically they are not secrets but agreed, I would avoid if I could. Unfortunately, those come from the Data Factory Git integration that "auto-commits" them in order to publish...

Reply:
Ok, that crossed my mind. Just wanted to make sure.

Reply:
I am not getting why do we need actual values if we are overriding things before the deployment? I think that this bit is confusing and should be explained in "README.md".

Reply:
I've just added to the readme :)